tests/wakebox/timeout-exceeded/input.json

```
@@ -0,0 +1,17 @@
+{
+  "command": [
+    "/bin/sh", "-c", "sleep 2 && true"
+  ],
```

One of the motivations for the timeout is that any already-written files are tracked by the fuse workspace even when the timeout otherwise halts progress.

We could check this like

```suggestion
  "command": [
    "/bin/sh", "-c", "touch hello; sleep 2 && true"
  ],
  "mount-ops": [
      {
        "type": "workspace",
        "destination": "."
      }
   ],

```
and then seeing if `hello` is in the outputs file

---

> and then seeing if `hello` is in the outputs file

Before I resolve this, do you be if its in the outputs _key_ in the JSON? Because it doesn't seem to be. But the file does exist/get created

---

oh I've just commented on the other file, yeah the thing we're looking for is that the json key for the outputs is populated

---

When I use this spec:

```
$ cat spec.json
{
  "command": [
    "/usr/bin/bash", "-c", "echo hello > yeah; sleep 10"
  ],
  "command_timeout": 9,
  "isolate-pids": true,
  "mount-ops": [
      {
        "type": "workspace",
        "destination": "."
      }
   ]
}
```
I see a file written:
```
$ ./bin/wakebox -p spec.json -o result.json
$ cat result.json | jq -r .outputs[]
yeah
```

---

got it. Alright, I'll have to poke around more. Probably won't have another commit for today as the day is wrapping up and someone is coming to work on the basement.

---

@JakeSiFive lmk if you have any ideas on what might be going wrong here